model/src/test/scala/explore/model/arb/ArbKeyedIndexedList.scala

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.model.arb
+
+import org.scalacheck.Arbitrary
+import org.scalacheck.Arbitrary._
+import org.scalacheck.Cogen
+import org.scalacheck.Cogen._
+import explore.data.KeyedIndexedList
+
+trait ArbKeyedIndexedList {
+  implicit def keyedIndexedListArb[K, A](implicit
+    arbA:     Arbitrary[A],
+    arbKeyFn: Arbitrary[A => K]
+  ) =
+    Arbitrary[KeyedIndexedList[K, A]] {
+      for {
+        list   <- arbitrary[List[A]]
+        getKey <- arbitrary[A => K]
```

Comment:
Do you get this arbitrary from a `Cogen`?

Reply:
Scalacheck materializes an `Arbitrary[A => K]` if it has an `Arbitrary[K]` and `Cogen[A]` in scope.